docs/user-guide/cli-using-understand-profiles-configs.md

```diff
@@ -102,11 +102,11 @@ The table below shows how Zowe CLI determines which profiles, properties, and va
+| <ul><li> global user profile </li> <li> global team profile </li></ul> | <ul><li> When the same property exists within the same profile in both config files, the property value from the global user config is used. </li> <li> When the same profile exists in both config files, but a property of that profile exists in only one file, that property is used. </li> <li> If a profile exists in only one config file, that profile is used in its entirety.</li> </ul> | **abc:** *direction:* south <br/> **abc:** *numbers:* 123 <br/> **def:** *shape:* circle <br/> **ghi:** *texture:* bumpy <br/> **pqr:** *distance:* near |
+| <ul><li> project team profile </li> <li> global user profile </li> <li> global team profile </li> </ul>| <ul><li> When a profile exists in all three config files, the project team profile is used.<sup>*</sup> </li> <li> If a profile exists in only one config file, that profile is used in its entirety. </li> </ul> | **abc:** *direction:* east <br/> **def:** *shape:* square <br/> **ghi:** *texture:* bumpy <br/> **mno:** *fruit:* banana <br/> **pqr:** *distance:* near |
+| <ul><li> project user profile </li> <li> project team profile </li> <li> global user profile </li> <li> global team profile </li> </ul> | <ul><li> When the same profile with the same properties exists in all four config files, the property values from the project user config is used. </li> <li> When the same profile exists in all four config files, the project files override the global files. If a property of the profile exists in only one of the two project configurations, that property is used.<sup>*</sup> </li> <li> If a profile exists in only one config file, that profile is used in its entirety. </li> </ul> |  **abc:** *direction:* north <br/> **def:** *shape:* triangle <br/> **ghi:** *texture:* bumpy <br/> **jkl:** *temperature:* cold <br/> **mno:** *fruit:* banana <br/> **pqr:** *distance:* near |
 
-<b>*</b> If the same profile exists in both a global configuration file and a project configuration file, the project configuration profile completely replaces the global profile. This is true even when the project profile has fewer properties in the same profile found in the global file.<br/><br/>
+<b>*</b> If the same profile exists in both a global configuration file and a project configuration file, the project configuration profile completely replaces the global profile. This is true even when the project profile has fewer properties in the same profile found in the global file.<br/>
 
 The rules above apply when profiles have the same name. To maintain the same set of properties in two different profiles, give each profile a different name so that Zowe CLI uses a specific profile, if needed.
```

Comment:
Replace "above" with "the previous rules" or move this line prior to the rules and say something like "the following rules."